src/components/fx/layout_attributes.js

```diff
@@ -29,38 +35,34 @@ module.exports = {
         valType: 'enumerated',
         role: 'info',
         values: ['x', 'y', 'closest', false],
+        editType: 'domodebar',
         description: 'Determines the mode of hover interactions.'
     },
 
     hoverlabel: {
         bgcolor: {
             valType: 'color',
             role: 'style',
+            editType: 'none',
```

Comment:
Interesting. This means a round of `supplyDefaults` is enough?

Reply:
`supplyDefaults`, plus `Plots.rehover` that also always gets called on edits.